test/evergreen.yml

```diff
@@ -358,6 +358,17 @@ functions:
           cd build_posix
           ${test_env_vars|} ${make_command|make} VERBOSE=1 check ${smp_command|} 2>&1
         fi
+
+  "cppsuite test":
+   - command: shell.exec
+     params:
+       working_dir: "wiredtiger/build_posix/"
+       script: |
+         set -o errexit
+         set -o verbose
+
+         ${test_env_vars|} $(pwd)/test/cppsuite/run -t ${test_name} -C '${test_config}' -f test/cppsuite/configs/${test_config_filename} -l 2
```

Comment:
I see that a few tests do not require the config arg. In that case -C equals '', is it similar to have no -C arg?

Reply:
Yes, I checked and it is the same.